content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+You need a working
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} on
+each Node in your cluster, so that the
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} can launch
+{{< glossary_tooltip text="Pods" term_id="pod" >}} and their containers.
+
+{{< glossary_definition term_id="container-runtime-interface" length="all" >}}
+
+The CRI is a plugin interface which enables the kubelet to use a wide variety of
```

Comment:
perhaps make this sentence first and one about the need for working container runtime second?

Reply:
Switched them as suggested :+1: